CMP1903_A1_2324/Testing.cs

```diff
@@ -15,6 +16,39 @@ internal class Testing
          * Use debug.assert() to make the comparisons and tests.
          */
 
+        /// <summary>
+        ///  The testing class rolls 3 dice and checks if the following is true:
+        ///  Every roll is between the values 1 and 6
+        ///  The sum of rolls is between the values 3 and 18
+        /// </summary>
+
         //Method
+        public void Main()
+        {
+            Game testGame = new Game();
+            var testOutputs = testGame.Main();
+
+            int roll1 = testOutputs.Item1;
+            int roll2 = testOutputs.Item2;
+            int roll3 = testOutputs.Item3;
+            int sum = testOutputs.Item4;
+
+            // First dice testing
+            Debug.Assert(roll1 <= 6, "First dice roll is greater than 6");
```

Comment:
This could be further improved by running on one line using "&&"

Reply:
I agree with Jack, the use of and will decrease the amount of repeated lines.

Reply:
Decided to keep these lines separate to provide more information about the errors